staging/src/k8s.io/controller-manager/pkg/leadermigration/migrator_test.go

```diff
@@ -55,6 +55,22 @@ func TestLeaderMigratorFilterFunc(t *testing.T) {
 			},
 		},
 	}
+	wildcardConfig := &internal.LeaderMigrationConfiguration{
+		ResourceLock: "leases",
+		LeaderName:   "cloud-provider-extraction-migration",
+		ControllerLeaders: []internal.ControllerLeaderConfiguration{
```

Comment:
`ControllerLeaders: migrationconfig.DefaultLeaderMigrationConfiguration()` for brevity?

Reply:
This may change in a future release. Better leave it as is in tests.